src/theme/ods-chart-theme.ts

```diff
@@ -206,37 +188,43 @@ const THEMES: {
   [mode in ODSChartsMode]: {
     common: any;
     linesAxis: any;
-    categoricalColors: {
-      [colorSet in ODSChartsCategoricalColorsSet]: { color: string[] };
+    colors: {
+      [colorSet in ODSChartsColorsSet]: { color: string[] };
     };
-    sequentialColors: {
-      [colorSet in ODSChartsSequentialColorsSet]: { visualMapColor: string[] };
+    visualMapColors: {
+      [colorSet in ODSChartsColorsSet]: { visualMapColor: string[] };
     };
     linesStyle: { [style in ODSChartsLineStyle]: any };
   };
 } = {
   light: {
     common: LIGHT_COMMON,
     linesAxis: LIGHT_LINES_AXIS,
-    categoricalColors: {
-      default: LIGHT_CATEGORICAL_COLORS,
-      categorical: LIGHT_CATEGORICAL_COLORS_CATEGORICAL,
-      functional: LIGHT_CATEGORICAL_COLORS_FUNCTIONAL,
-      supportingColors: LIGHT_CATEGORICAL_COLORS_SUPPORTING_COLORS,
-      darkerTints: LIGHT_CATEGORICAL_COLORS_DARKER_TINTS,
-      lighterTints: LIGHT_CATEGORICAL_COLORS_LIGHTER_TINTS,
-      blue: LIGHT_CATEGORICAL_COLORS_BLUE,
-      green: LIGHT_CATEGORICAL_COLORS_GREEN,
-      pink: LIGHT_CATEGORICAL_COLORS_PINK,
-      purple: LIGHT_CATEGORICAL_COLORS_PURPLE,
-      yellow: LIGHT_CATEGORICAL_COLORS_YELLOW,
+    colors: {
+      default: LIGHT_COLORS,
+      categorical: LIGHT_COLORS_CATEGORICAL,
+      functional: LIGHT_COLORS_FUNCTIONAL,
+      supportingColors: LIGHT_COLORS_SUPPORTING_COLORS,
+      darkerTints: LIGHT_COLORS_DARKER_TINTS,
+      lighterTints: LIGHT_COLORS_LIGHTER_TINTS,
+      blue: LIGHT_COLORS_BLUE,
+      green: LIGHT_COLORS_GREEN,
+      pink: LIGHT_COLORS_PINK,
+      purple: LIGHT_COLORS_PURPLE,
+      yellow: LIGHT_COLORS_YELLOW,
     },
-    sequentialColors: {
-      blue: LIGHT_SEQUENTIAL_COLORS_BLUE,
-      green: LIGHT_SEQUENTIAL_COLORS_GREEN,
-      pink: LIGHT_SEQUENTIAL_COLORS_PINK,
-      purple: LIGHT_SEQUENTIAL_COLORS_PURPLE,
-      yellow: LIGHT_SEQUENTIAL_COLORS_YELLOW,
+    visualMapColors: {
+      default: { visualMapColor: LIGHT_COLORS.color },
```

Comment:
I can't remember what we've said to do for the visual map colors, do we want all the colors ?

Reply:
I don't think we've decided anything. 

In the first version, I chose to include only color palettes of the same color. 

But now that we've simplified things by mutualizing color management, I don't see much reason to restrict it. Let's keep it simple, shall we?

Reply:
You're right :)